fiftyone/operators/executor.py

```
@@ -492,7 +523,10 @@ async def resolve_execution_options(registry, operator_uri, request_params):
     )
     await ctx.resolve_secret_values(operator._plugin_secrets)
     try:
-        return operator.resolve_execution_options(ctx)
+        # User code
+        with ctx:
+            execution_options = operator.resolve_execution_options(ctx)
```

Pylint (4.0.2)

[warning] Unused variable 'execution_options'

(W0612)